django/db/models/query.py

```diff
@@ -495,8 +494,7 @@ def _create_object_from_params(self, lookup, params):
         Used by get_or_create and update_or_create
         """
         try:
-            with transaction.atomic(using=self.db):
-                obj = self.create(**params)
+            obj = self.create(**params)
```

Comment:
I think this one requires a savepoint, but create() calls save(), and save has savepoint=False.